src/components/common/organisms/ChangeLog.tsx

```diff
@@ -14,26 +21,25 @@ const ChangeLog = (props: {
   }) => {
 
   const renderChangeLog = (release : any) => {
-    const { changelogItem, subtitleHeader, changelogText, bullet, bulletList } = styles;
-
-    const changes = release.changes || [];
+    const { changelogItem, versionHeader, subtitleHeader } = styles;
+    const html = {html: converter.makeHtml(release.releaseNote)};
+    const date = moment(release.releaseDate).format('MMM DD, YYYY');
     return (
       <ScrollView style={changelogItem}>
-        <Text style={subtitleHeader}>{release.header}</Text>
-        <Text style={changelogText}>{release.subHeader}</Text>
-        <View style={{ margin: 5, marginLeft: 10 }}>
-          {changes.map((change: boolean | React.ReactChild | React.ReactFragment | React.ReactPortal | null | undefined, index: { toString: () => React.Key | null | undefined; }) => (
-            <View key={index.toString()} style={bulletList}>
-              <Text style={bullet}>{'\u2022'}</Text>
-              <Text style={changelogText}>{change}</Text>
-            </View>
-          ))}
+        <Typography style={versionHeader} medium={true}>{release.version}</Typography>
+        <Typography style={subtitleHeader} medium={true}>{date}</Typography>
+        <View>
+          <RenderHtml
+            contentWidth={useWindowDimensions().width}
```

Comment:
Most of the web apps split large screens into the main view and a backdrop:
<img width="1596" alt="Screenshot 2022-12-16 at 10 08 56" src="https://user-images.githubusercontent.com/36899138/208063416-ac7a5069-2f7b-4148-a5a6-b0149fc646d4.png">

I am assuming using the window width could cause issues for these apps?
I had a similar issue some weeks ago and found out that [`<View>`](https://reactnative.dev/docs/view#onlayout) has an `onLayout` prop that can be used to fetch the view dimensions rather than the whole screen dimensions. Maybe this is something that needs to be looked at

Reply:
changed to use onLayout